docs/application/toc_all.md

```diff
@@ -142,6 +142,7 @@
 ##### [Overview](/application/dotnet/guides/multimedia/media-vision.md)
 ##### [Barcode Detection and Generation](/application/dotnet/guides/multimedia/image-barcode.md)
 ##### [Face Detection, Recognition, and Tracking](/application/dotnet/guides/multimedia/face-detection.md)
+##### [Deep Learning Based Face Recognition](/application/dotnet/guides/multimedia/face-recognition.md)
```

Comment:
I think it should be placed in the Table of Contents according to the alphabetical order.
```suggestion
##### [Barcode Detection and Generation](/application/dotnet/guides/multimedia/image-barcode.md)
##### [Deep Learning Based Face Recognition](/application/dotnet/guides/multimedia/face-recognition.md)
##### [Face Detection, Recognition, and Tracking](/application/dotnet/guides/multimedia/face-detection.md)
```